docs/pages/collection/[collectionAddress].tsx

```diff
@@ -155,13 +133,8 @@ export default function CollectionPage({
         <div className="nft-container mt-5">
           {nftsInCollection.map((nft) => (
             <Link href={`/nft/${nft.address}`} key={nft.address}>
-              <a className="nft">
-                {nft.image && (
-                  <div className="image">
-                    <SquareImage src={nft.image} size={400} alt={nft.name} />
-                  </div>
-                )}
-                <div className="name">{nft.name}</div>
+              <a className="nft-card-link">
```

Comment:
```suggestion
              <LuxLink className="nft-card-link">
```

If we use `LuxLink` then we don't need to have two components for every link cc @benborgers